website/pages/docs/job-specification/service.mdx

```diff
@@ -705,7 +705,7 @@ advertise and check directly since Nomad isn't managing any port assignments.
 </sup>
 <small>
   {' '}
-  Script checks are not supported for the [qemu driver][qemu] since the Nomad
+  Script checks are not supported for the [qemu] driver since the Nomad
```

Comment:
I'm looking at the preview and I don't think the link rendering is working at all in this `<small>` block. Take a look at the end of this page: https://nomad-evt8hfzhb.vercel.app/docs/job-specification/service

It looks like this is a bug on the existing page too. If you know how to fix that, great. Otherwise maybe we just drop that link entirely?

Reply:
This is how I ended up suggesting an additional inline link on this page.  I'm not up-to-speed with mdx yet but will be starting on Learn content in due course so will be able to make more effective PRs in future.  I'll step back from this PR as a result, but will monitor. Thanks @tgross

Reply:
@jescalan do you have any insights here as to why a valid link wouldn't be rendering in this context?

Reply:
Yes, html and markdown cannot be mixed currently within mdx. If you open up a block of html, anything inside of it must be html until the close of the block. If you make this link use an `<a>`, it should work fine!

Reply:
Ok, in which case let's rebase this on master (because there are merge conflicts) and remove the link entirely.